src/BaGet.Protocol/BaGet.Protocol.csproj

```diff
@@ -11,7 +10,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="$(MicrosoftExtensionsPackageVersion)" />
     <PackageReference Include="NuGet.Frameworks" Version="$(NuGetPackageVersion)" />
     <PackageReference Include="NuGet.Versioning" Version="$(NuGetPackageVersion)" />
-    <PackageReference Include="System.Text.Json" Version="4.7.2" />
+    <PackageReference Include="System.Text.Json" Version="6.0.1" />
```

Comment:
We can remove this line now that this project targets .NET 6 as that includes `System.Text.Json` in the standard library

```suggestion
```

Reply:
Done